Tests/Modules/MetadataModuleSpecs.swift

```diff
@@ -132,7 +132,7 @@ class MetadataModuleSpecs: QuickSpec {
                                         ["key": "FY14"],
                                         ["key": "FY15"]
                                     ],
-                                    "key": "fy", "displayName": "FY"]
+                                     "key": "fy", "displayName": "FY"]
```

Comment:
minor: unnecessary extra space before "key"?

Reply:
A lot of the small changes happened when I ran the tests. Seems like the linter did it.